src/view/use_state.rs

```diff
@@ -103,3 +108,12 @@ where
         a
     }
 }
+
+pub fn use_state<T, A, S, V, FInit, F>(f_init: FInit, f: F) -> UseState<T, A, S, V, FInit, F>
```

Comment:
Can you add a doc comment here?

Reply:
If I would understand exactly what `UseState` does, I'd do ^^ (see zulip thread). I think probably someone else should document the code (maybe in another PR), this function just runs `UseState::new` obviously, I added the trait bounds, so that it's easier to see in rust-analyzer what is required to get this into a `View` (as it otherwise doesn't make much sense I think).

Reply:
Added a comment, could probably be better, but I guess better than nothing for now.